src/main/java/art/heredium/domain/membership/entity/MembershipRegistrationHistory.java

```diff
@@ -0,0 +1,72 @@
+package art.heredium.domain.membership.entity;
+
+import java.time.LocalDate;
+
+import javax.persistence.*;
+
+import lombok.*;
+
+import com.vladmihalcea.hibernate.type.json.JsonStringType;
+import org.hibernate.annotations.DynamicInsert;
+import org.hibernate.annotations.TypeDef;
+
+import art.heredium.domain.account.entity.Account;
+
+@Entity
+@Getter
+@Table(name = "membership_registration_history")
```

Comment:
```suggestion
@Table(name = "company_membership_registration_history")
```
Để đỡ bị lẫn với luồng user tự đăng ký membership. Bảng này chỉ nhằm mục đích lưu lại history của việc upload excel cho company thôi.

Reply:
Tôi đã fix: a47b74dc6e231a1aff35f9de573803e39d437540